### DISCUSSION TITLE
Some mods can't be found in the questcraft launcher

### DISCUSSION THREAD

**Discussion by Zircon10086** [QuestCraftPlusPlus/QuestCraft#581] [Q&A] [answered]
When I search for some mods in the questcraft, sometimes I can't find it. But another website has the mod.
And the game keeps crashing when I try to move the mod file (jar) in
`/storage/emulated/0/Android/data/com.qcxr.qcxr/files/instances/1.20.4/mods/ `
by using a file manager.

This is the mod, which can't be found in questcraft.
[https://www.mcmod.cn/class/12636.html](url)

I'm glad to see that in the future, players will be able to import their own mods, rather than searching for them in Questcraft, because there are always some interesting mods that cannot be found through searching.

**Comment by KendrickLamar2003** [comment]
So it appears that this mod simply cannot work with the game. Sorry mate, but until some big compatibility update comes to QuestCraft, emergency food is simply not possible.

**Reply by KendrickLamar2003** [reply, marked as answer]
Alright, here is an update: The mod does need a dependency called trinkets, however, even adding it still makes the game crash.

**Reply by Zircon10086** [reply]
Thanks! Maybe I required for too much, but I do hope that more mods can be added freely in the future, not only the Emergency Food.
